app/enterprise/1.5.x/kong-for-kubernetes/index.md

```diff
@@ -1,14 +1,17 @@
 ---
 title: Kong for Kubernetes Enterprise
+toc: false
 ---
 
-Kong for Kubernetes Enterprise (K4K8S) is a Kubernetes Ingress Controller. A Kubernetes Ingress Controller is a proxy that exposes Kubernetes services from applications (e.g., Deployments, RepliaSets) running on a Kubernetes cluster to client applications running outside of the cluster. The intent of an Ingress Controller is to provide a single point of control for all incoming traffic into the Kubernetes cluster.
+Kong for Kubernetes Enterprise is a deployment of {{site.ee_gateway_name}} onto Kubernetes with an ingress controller, which is responsible for configuring Kong. A Kubernetes ingress controller is a proxy that exposes Kubernetes services from applications (e.g., Deployments, ReplicaSets) running on a Kubernetes cluster to client applications running outside of the cluster. The intent of an ingress controller is to provide a single point of control for all incoming traffic into the Kubernetes cluster.
```

Comment:
I'm not sure how to best describe this effectively, but the first sentence indicates that the controller is the controller process alone and the second indicates it's the proxy (or the complete unit). Maybe:

> Kong for Kubernetes Enterprise is a deployment of {{site.ee_gateway_name}} onto Kubernetes as an ingress controller. A Kubernetes ingress controller ...

We probably don't need to make the distinction between the components here; https://docs.konghq.com/enterprise/1.5.x/kong-for-kubernetes/using-kong-for-kubernetes/#architecture links to a more detailed explanation.

StatefulSets may be a better example than ReplicaSets (normally you don't create the latter directly), but that doesn't really matter.

Reply:
It's an easy enough change, so might as well :)

For the current description: I edited it the way you suggested, and I think the rest doesn't go into too much detail to be a worry. Will link off to the architecture doc as well.